types/field_type.go

```diff
@@ -39,7 +39,8 @@ type FieldType struct {
 	Charset string
 	Collate string
 	// Elems is the element list for enum and set type.
-	Elems []string
+	Elems           []string
+	BelowZeroBeZero bool
```

Comment:
we need this... because we can not cast < 0 to zero directly in chunk_executor...- -

because `select cast(-1 as unsigned)` need be MAX_UINT64..
but for union or insert need be 0